src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,25 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        if (email.equals("bob@i.ua") && password.equals("1234")) {
+
+            return true;
+        }
+        if (email.equals("alice@i.ua") && password.equals("1234")) {
+
+            return true;
+
+        } else {
+            return false;
+        }
```

Comment:
it's a hardcoding, it's not ok. You should create an instance of UserService here and call findByEmail method to check if user can login with such email and password